_includes/payee-info.md

```diff
@@ -1,4 +1,5 @@
 {% assign instrument = include.payment_instrument | default: "paymentorder" %}
+{% assign invoice_payments = include.invoice_payments | default: false%}
```

Comment:
This can be set to the evaluation of `payment_instrument == "invoice"` and not be required as an include parameter.